package.json

```diff
@@ -26,16 +26,16 @@
   },
   "homepage": "https://github.com/mixmaxhq/rewt#readme",
   "dependencies": {
+    "jest": "^26.5.3",
     "jsonwebtoken": "^8.3.0",
     "node-redis-scripty": "0.0.5",
+    "promise-callbacks": "^3.8.2",
```

Comment:
Can this be a dev dependency? (Same with jest)

Reply:
Doh, this is what I get for not having enough coffee yet today. Thanks for the catch!